src/main/java/Commands.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Commands {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static int numberOfTasks = 0;
+    public static int completedTasks = 0;
+    public static int remainingTasks;
+    public static final Scanner scan = new Scanner(System.in);
```

Comment:
Constant (final) should be named in all caps. e.g. SCAN

Reply:
Thank you for your feedback. I have corrected the mistake.